core/src/main/java/uk/gov/justice/services/core/json/DefaultJsonSchemaValidatorFactory.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.justice.services.core.json;
+
+import org.slf4j.LoggerFactory;
+
+public class DefaultJsonSchemaValidatorFactory {
```

Comment:
Shouldn't this class be in Test not Main?

Reply:
Moved to test-core